### DISCUSSION TITLE
Error while installing Ganache through Windows Installer - Error Code 0x8000FFFF

### DISCUSSION THREAD

**Discussion by Jobo0o** [ConsenSys-archive/ganache#3324] [Q&A] [answered]
I'm getting a consistent error while trying to install the Ganache Client on Windows through the Windows Installer after downloading from the official website. See below the Error Code:
```
Deployment Add operation with target volume C: on Package GanacheUI_2.5.4.0_x64__5dg5pnz03psnj from:  (Ganache-2.5.4-win-x64%20(2).appx)  failed with error 0x8000FFFF. See http://go.microsoft.com/fwlink/?
```
The Microsoft Link at the end is not active anymore and I could not find the error occurring with others so far.
Is there any workaround for installing the client? Is this a known bug or has this something to do with my machine?

Thanks!

**Comment by apdan510** [comment, marked as answer]
@Jobo0o Try this installer, same version but in .exe format. https://github.com/trufflesuite/ganache-ui/releases/download/v2.5.4/Ganache-2.5.4-win-setup.exe

Hope this helps!
-Alex

**Reply by Jobo0o** [reply]
Thanks that worked out perfectly. Thank! 
I was looking for the .exe file before, but I could not find it in the Repo...
